Fiz o teste funcional aqui e para a emissão de NF-e e não está funcionando:

```
Traceback (most recent call last):
  File "/opt/odoo/auto/addons/queue_job/controllers/main.py", line 107, in runjob
    self._try_perform_job(env, job)
  File "/opt/odoo/auto/addons/queue_job/controllers/main.py", line 37, in _try_perform_job
    job.perform()
  File "/opt/odoo/auto/addons/queue_job/job.py", line 520, in perform
    self.result = self.func(*tuple(self.args), **self.kwargs)
  File "/opt/odoo/auto/addons/l10n_br_fiscal_queue/model/document.py", line 24, in _send_document_job
    record._eletronic_document_send()
  File "/opt/odoo/auto/addons/l10n_br_nfse_paulistana/models/document.py", line 326, in _eletronic_document_send
    super(Document, self)._eletronic_document_send()
  File "/opt/odoo/auto/addons/l10n_br_nfe/models/document.py", line 947, in _eletronic_document_send
    record.atualiza_status_nfe(processo)
  File "/opt/odoo/auto/addons/l10n_br_nfe/models/document.py", line 887, in atualiza_status_nfe
    file_response_xml=processo.envio_xml.decode("utf-8"),
AttributeError: 'str' object has no attribute 'decode'
```

O teste unitário também não está passando por outro motivo:

```
2023-08-11 11:58:04,208 311 ERROR odoo odoo.addons.l10n_br_nfe.tests.test_nfce: ERROR: TestNFCe.test_atualiza_status_nfce
Traceback (most recent call last):
  File "/__w/l10n-brazil/l10n-brazil/l10n_br_nfe/tests/test_nfce.py", line 222, in test_atualiza_status_nfce
    self.document_id.atualiza_status_nfe(mock_autorizada)
  File "/__w/l10n-brazil/l10n-brazil/l10n_br_nfe/models/document.py", line 873, in atualiza_status_nfe
    file_response_xml=processo.envio_xml.decode("utf-8"),
  File "/usr/lib/python3.8/unittest/mock.py", line 637, in __getattr__
    raise AttributeError("Mock object has no attribute %r" % name)
AttributeError: Mock object has no attribute 'envio_xml'
```

--- a/l10n_br_nfe/__manifest__.py
+++ b/l10n_br_nfe/__manifest__.py
@@ -41,7 +41,7 @@
             "nfelib>=2.0.0",
             "erpbrasil.assinatura>=1.7.0",
             "erpbrasil.transmissao",
-            "erpbrasil.edoc>=2.5.1",
+            "erpbrasil.edoc>=2.5.2",
             "erpbrasil.edoc.pdf",
         ],
     },

--- a/l10n_br_nfe/models/document.py
+++ b/l10n_br_nfe/models/document.py
@@ -870,7 +870,7 @@ class NFe(spec_models.StackedModel):
                 response=infProt.xMotivo,
                 protocol_date=protocol_date,
                 protocol_number=infProt.nProt,
-                file_response_xml=processo.envio_xml.decode("utf-8"),
+                file_response_xml=processo.processo_xml.decode("utf-8"),
             )
         self.write(
             {

--- a/l10n_br_nfe/tests/test_nfce.py
+++ b/l10n_br_nfe/tests/test_nfce.py
@@ -218,7 +218,7 @@ class TestNFCe(TestNFeExport):
         mock_autorizada.protocolo.infProt.cStat = AUTORIZADO[0]
         mock_autorizada.protocolo.infProt.xMotivo = "TESTE AUTORIZADO"
         mock_autorizada.protocolo.infProt.dhRecbto = datetime.now()
-        mock_autorizada.envio_xml = b"dummy"
+        mock_autorizada.processo_xml = b"dummy"
         self.document_id.atualiza_status_nfe(mock_autorizada)
 
         self.assertEqual(self.document_id.state_edoc, SITUACAO_EDOC_AUTORIZADA)

--- a/requirements.txt
+++ b/requirements.txt
@@ -4,7 +4,7 @@ erpbrasil.assinatura>=1.7.0
 erpbrasil.base>=2.3.0
 erpbrasil.edoc
 erpbrasil.edoc.pdf
-erpbrasil.edoc>=2.5.1
+erpbrasil.edoc>=2.5.2
 erpbrasil.transmissao
 nfelib>=2.0.0
 nfselib.ginfes

